AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -12,6 +12,7 @@ ligainsider.de###popup_extra
 kickassanime.io###notes
 ! facebook.com - reactions in live-video even when disabled via button
 facebook.com###fbVideoStreamingReactionsTray
+kompkimi.ru###print-content > .box_author > .box-option > .p_bold
```

Comment:
why you block that element?
![image](https://user-images.githubusercontent.com/8361299/61872805-edfff400-aeec-11e9-9055-81431b5f5683.png)

Reply:
@Alex-302, removed redundant rule